frontend/public/models/yaml-templates.ts

```diff
@@ -767,4 +767,43 @@ spec:
     delayAfterAdd: 10s
     delayAfterDelete: 10s
     delayAfterFailure: 10s
+`).setIn([referenceForModel(k8sModels.VirtualMachineModel), 'default'], `
```

Comment:
Can we move this code somewhere under `kubevirt` directory?
Like export a function there which will return content to be passed here.

Just to stay aligned with the rule about minimal modifications to the shared code to avoid merge conflicts or at least simplify their resolving.

Reply:
@mareklibra This was done in the same way as with k8s models. We didn't import anything in `public/models/index.ts` - we just added KubeVirt related stuff directly into that file. The change you've proposed goes a bit against this principle, but I think it will change anyway before our stuff gets merged into `master` of openshift/console.

Reply:
I understand your point, in "normal" situation I would not rise that, it makes sense to keep consistency.

Anyway, we regularly (almost daily) merge upstream to our fork and every line of "our" change in the OKD-core source leads to merge conflicts. Due to that fact, we can't use automation for these upstream merges. So I want to make my life a little bit easier at least this way.
So far we carefully consider _every_ line we introduce in okd-core part and if possible to avoid it (even using non-fancy hacks).